project/scripts/processDocs.sc

```diff
@@ -0,0 +1,49 @@
+import java.nio.file.{ FileSystems, Files, Path }
+import java.io.PrintStream
+import scala.io.{ Codec, Source }
+import scala.collection.JavaConverters._
+
+val inputDir = FileSystems.getDefault.getPath("docs-for-scalalang")
```

Comment:
I think we keep docs in `docs` dir and process them into `docs-for-dotty-page` dir to avoid confusion.